helm/test.tmp.yaml

```diff
@@ -0,0 +1,372 @@
+kind: ClusterRole
+metadata:
+  # namespace: "myns"
+  name: wrongsecrets-cleaner
+  labels:
+    helm.sh/chart: wrongsecrets-ctf-party-0.1.0-alpha
+rules:
+  - apiGroups: ['apps']
+    resources: ['deployments']
+    verbs: ['get', 'delete', 'list']
+  - apiGroups: [''] # "" indicates the core API group
+    resources: ['services']
+    verbs: ['get', 'delete']
+  - apiGroups: [''] # "" indicates the core API group
+    resources: ['namespaces']
+    verbs: ['get', 'delete', 'list']
+---
+# Source: wrongsecrets-ctf-party/templates/wrongsecrets-balancer/role.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: wrongsecrets-balancer
+  labels:
+    helm.sh/chart: wrongsecrets-ctf-party-0.1.0-alpha
+rules:
+  - apiGroups: ['apps']
+    resources: ['deployment', 'deployments']
+    verbs: ['get', 'create', 'list', 'delete', 'patch', 'watch', 'update']
+  - apiGroups: [''] # "" indicates the core API group
+    resources: ['services']
+    verbs: ['get', 'list', 'create', 'delete']
+  - apiGroups: [''] # "" indicates the core API group
+    resources: ['pod', 'pods', 'pods/log']
+    verbs: ['create', 'get', 'list', 'delete', 'watch']
+  - apiGroups: [''] # "" indicates the core API group
+    resources: ['namespaces']
+    verbs: ['get', 'create', 'list', 'delete', 'patch', 'watch', 'update']
+  - apiGroups: [''] # "" indicates the core API group
+    resources: ['configmaps']
+    verbs: ['create', 'get', 'list', 'delete']
+  - apiGroups: [''] # "" indicates the core API group
+    resources: ['secrets']
+    verbs: ['create', 'get', 'list', 'delete']
+  - apiGroups: ['']
+    resources: ['serviceaccounts']
+    verbs: ['create', 'get', 'list', 'delete', 'patch', 'update']
+  - apiGroups: ['rbac.authorization.k8s.io']
+    resources: ['roles']
+    verbs: ['create', 'delete', 'deletecollection', 'get', 'list', 'patch', 'update', 'watch', 'admin', 'escalate']
+  - apiGroups: ['rbac.authorization.k8s.io']
+    resources: ['rolebindings']
+    verbs: ['create', 'get', 'list', 'delete', 'patch', 'grant', 'bind', 'escalate']
+  - apiGroups: ['rbac.authorization.k8s.io']
+    resources: ['roleRef']
+    verbs: ['create', 'get', 'list', 'delete', 'patch', 'grant', 'escalate']
+  - apiGroups: ['secrets-store.csi.x-k8s.io']
+    resources: ['secretproviderclasses']
+    verbs: ['create', 'get', 'list', 'delete']
+  - apiGroups: ['networking.k8s.io']
+    resources: ['networkpolicies']
+    verbs: ['create', 'get', 'list', 'delete']
+  - apiGroups: ['']
+    resources: ['endpoints']
+    verbs: [ 'get', 'list']
+---
+# Source: wrongsecrets-ctf-party/templates/cleanup/rolebinding.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: wrongsecrets-cleaner
+  # namespace: "myns"
+  labels:
+    helm.sh/chart: wrongsecrets-ctf-party-0.1.0-alpha
+subjects:
+  - kind: ServiceAccount
+    name: wrongsecrets-cleaner # Name is case sensitive
+    namespace: "myns"
```

Comment:
Not sure if we need this? How is it used?

Reply:
I think the service account is needed for the `balancer` deployment to spin up more deployment instances of the `wrongsecrets` in case of need. So, it is required to be able to execute k8s commands
Is it how it works or does the balancer have another mechanism so it doesn't need it??

Reply:
ah no: I mean: why do we have the namespace "myns"? (yes we need the service account :D )

Reply:
Actually, it was just overwritten by the helm template command in the `package.json`
https://github.com/OWASP/wrongsecrets-ctf-party/pull/249/files#diff-f082820c884087266993cefee991563d78eee3d159536b130576c9985e5b9061R11
It will not be affected otherwise in real `helm apply`